book/src/user/requirements.md

```diff
@@ -1,16 +1,22 @@
 # System Requirements
 
-We recommend the following requirements for compiling and running `zebrad`:
+Zebra has the following hardware requirements.
+
+## Recommended Requirements
```

Comment:
Subjective/Nitpick/Optional:

```suggestion
## Recommended Requirements
```

Reply:
I prefer not to have an empty space between section titles and subtitles, similarly to the two folks who answered this question: https://tex.stackexchange.com/questions/108843/if-section-is-followed-by-a-subsection-should-there-be-nothing-in-between.